modules/serverless-config-replicas-kafka.adoc

```diff
@@ -6,7 +6,7 @@
 [id="serverless-config-replicas-kafka_{context}"]
 = Configuring high availability replicas for Knative Kafka
 
-You can scale Knative Kafka components by modifying the `spec.high-availability.replicas` value in the `KnativeKafka` custom resource (CR).
+High availability (HA) is available by default for the Knative Kafka `kafka-ch-controller`, `kafka-controller-manager`, and `kafka-webhook` components, which are configured to have two each replicas by default. You can change the number of replicas for these components by modifying the `spec.high-availability.replicas` value in the `KnativeKafka` custom resource (CR).
```

Comment:
We need to align this with this change: https://github.com/openshift/openshift-docs/pull/45935/files#diff-f063d1a1aa46ff35cb8af53f5413d2dca4703db1e1777ae27b9e86f611fc3615R25
Otherwise the rest of this PR looks good.